chats/migrations/0001_initial.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 5.0.4 on 2024-05-08 07:43
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ChatGroup',
```

Comment:
```suggestion
            name='ChatGroup',
```

程式碼改成像這樣縮排 `initial = True`，統一方便閱讀

Reply:
好的！